### PR TITLE
SEAR-241: fix LIS facility assigned worker contact information and change HTTP response status 

### PR DESCRIPTION
**Pull request by pradipforever**
1. SEAR-241: fix LIS facility assigned worker contact information
2. change HTTP response status code to 200 for empty children collection

### JIRA Issue Link
- https://osi-cwds.atlassian.net/browse/CALS-NNNN

### Technical Description
First Change: Change the facility mapper DTO to get phone number of LpaTelephone instead of LpaDoPhone

Second Change: Returning HTTP response 200 instead of 404: Returned empty collection from rest API instead of null, changed tests from NotFoundException and 404 to empty collection and 200. 

### Tests
- [X] I have included unit tests 
- [ ] I have included integration tests 
- [ ] I have included performance tests 
- [ ] I have included other tests 
- [ ] I have NOT included tests 

### Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by LeonidMarushevskyi** [comment]
Jenkins retest this please

**Comment by pradipforever** [comment]
retest this please

File: src/test/java/gov/ca/cwds/cals/web/rest/facility/FacilityChildAuthorizationTest.java [hunk 1]
```diff
@@ -28,9 +29,11 @@ public static void beforeClass() throws Exception {
     setUpCmsRs();
   }
 
-  @Test(expected = NotFoundException.class)
+    @Test
   public void testStateSealed() throws Exception {
-    getFacilityChildrenDTO("security/client/principal-state-sealed.json");
+        FacilityChildrenDto facilityChildrenDto = getFacilityChildrenDTO("security/client/principal-state-sealed.json");
+        // tests if the collection is empty instead of NotFoundException.class, children collection is empty, not null
```
Comment (AlexanderSerbin) [inline src/test/java/gov/ca/cwds/cals/web/rest/facility/FacilityChildAuthorizationTest.java]: Comment does not help. Please consider removing it

**Comment by pradipforever** [comment]
retest this please

**Pull request merged**